vuepress/docs/next/docs/getting-started/README.md

```diff
@@ -158,7 +158,7 @@ sudo kubectl create namespace entando
 #### Add Custom Resources
 
 ::: tip Why Custom Resources?
-Standard resources in Kubernetes include things like `Pods`, which are groups of one or more containers, `Services`, the way to call or access your pods, and `Ingresses`, for managing external access to your cluster.
+Standard resources in Kubernetes include things like `Pods`, which are groups of one or more containers, `Services`, the way to call or access pods, and `Ingresses`, for managing cluster resources.
```

Comment:
Sorry, we may have over-worked this one. Ingresses are for providing ways for external systems to connect to cluster resources. The original language looks better at the moment "for managing external access to your cluster". Ingresses don't manage cluster resources, they open a way to communicate with them...

Reply:
..."for managing cluster communications from external sources" or "to manage communications from outside the cluster" (i like the latter)? a variation of one of these?

Reply:
Or "to enable external access to services".  Not sure why I like it better but shorter and "enable" doesn't promise as much as "manage" somehow. Ingresses map incoming external requests to internal resources. Technically the access could be for something outside the namespace but still inside the cluster...

Reply:
i like yours - leaner and noncommital :) agreed "manage" implies a set of capabilities and/or oversight of access protocol